pldp/forms.py

```diff
@@ -106,9 +106,95 @@
     ('75+', _('75+')),
 ]
 
-# Choices for SurveyComponentMode
+# Choices for mode_observational question
 MODE_TYPE_CHOICES = [
     ('basic', _('Basic choices')),
     ('detailed', _('Detailed choices')),
-    ('complex', _('Complex choices')),
+]
+
+MODE_BASIC_CHOICES = [
+    ('pedestrian', _('Pedestrian')),
+    ('bicyclist', _('Bicyclist')),
+    ('animal', _('Moving by animal')),
+    ('snow', _('Moving in snow')),
+    ('water', _('Moving by water')),
+    ('vehicle', _('Moving by vehicle')),
+]
+
+MODE_DETAILED_CHOICES = [
+    ('pedestrian_walking', _('Pedestrian walking')),
```

Comment:
Does the PLDP provide any guidance on comparing detailed to basic choices? It strikes me that ‘pedestrian_walking’ is a subset of ‘pedestrian’, but there’s no way to know that without doing string manipulation in the app code. Maybe the solution is to have a dict in this module that maps detailed to basic choices — in any case, not a requirement for this PR, just something I’m thinking about.

Reply:
> NOTE: A public life study typically measures pedestrians and bicyclists. In low-density location, these categories may be subdivided into their subcategories to collect a greater level of detail. A survey can consist of a mix of main and subcategories.

There are several levels of subcategories and I'm not sure yet the best way to handle them with the MultiWidget. A dict would work!